### PR TITLE
Adding dvc-task

### PR DESCRIPTION
**Pull request by pmrowla**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pmrowla** [comment]
Pinging @efiop and @shcheklein for confirmation on maintainers list

**Comment by pmrowla** [comment]
Build issue is that dvc-task requires the latest kombu + celery release (`5.3.0.a1`) which is available on pypi but have not been released on conda yet:
https://github.com/conda-forge/kombu-feedstock
https://github.com/conda-forge/celery-feedstock

**Comment by shcheklein** [comment]
I confirm!

**Comment by pmrowla** [comment]
@conda-forge/help-python this is ready for review

File: recipes/dvc-task/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+    - python >=3.7
+    - setuptools_scm
+  run:
+    - celery
+    - funcy
+    - kombu
+    - python >=3.7
+    - pywin32-on-windows
+    - shortuuid
```
Comment (dopplershift) [inline recipes/dvc-task/meta.yaml:28]: Let's match the version ranges from upstream:
```suggestion
    - celery >=5.2.0,<6
    - funcy >=1.17
    - kombu
    - python >=3.7
    - pywin32-on-windows >=225
    - shortuuid >=1.0.8
```

Comment (pmrowla) [reply recipes/dvc-task/meta.yaml]: This is done (with the note that `pywin32-on-windows` is still unversioned and the `pywin32>=225` constraint is in `run_constrained`)

**Comment by pmrowla** [comment]
@conda-forge/help-python needs re-review

**Review by dopplershift — APPROVED** [review]

**Pull request merged**